src/server/app/util/services/inactivity-email.server.service.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+let _ = require('lodash'),
+	path = require('path'),
+	q = require('q'),
+	deps = require(path.resolve('./src/server/dependencies.js')),
+	handlebars = require('handlebars'),
+	fs = require('fs'),
+	dbs = deps.dbs,
+
+	configc = deps.config,
+	emailService = deps.emailService,
+	logger = deps.logger,
+	User = dbs.admin.model('User');
+
+
+function buildEmailContent(resource, emailTemplateName, config) {
+	let numOfDays = Math.floor((Date.now() - resource.lastLogin)/config.day);
+	let emailData = {
+		appName: configc.app.instanceName,
+		url: configc.app.url.protocol + '://' + configc.app.url.host + ':' + configc.app.url.port + '/#', // message of the day board
```

Comment:
Let's use the configc.app.baseUrl instead. Did you mean to fill in the url for message of the day?